translations/translations.json

```diff
@@ -5937,5 +5937,56 @@
     "zh-CN": {
       "noOffersAvailable": "目前没有提供项目"
     }
+  },
+  "shelter": {
+    "de": {
+      "title": "Wohnraum",
+      "shelterTitle": "{{beds}} in {{location}}",
+      "shelterInformation": "Verfügbarer Wohnraum",
+      "shelterButton": "Anschauen",
+      "additionalInformation": "Weitere Angaben zum Wohnraum",
+      "hostInformation": "Angaben zum Anbietenden",
+      "contactInformation": "Kontaktdaten",
+      "bed": "1 Bett",
+      "beds": "{{beds}} Betten",
+      "ges_unterkunft": "Gesamte Unterkunft",
```

Comment:
Not sure if this should have capital first letter.

Reply:
Imo yes, but I'll ask Daniel again.